kbfsgit/runner.go

```diff
@@ -232,56 +267,184 @@ func (r *runner) waitForJournal(ctx context.Context) error {
 	return nil
 }
 
-func (r *runner) handleList(ctx context.Context, args []string) error {
+// handleList: From https://git-scm.com/docs/git-remote-helpers
+//
+// Lists the refs, one per line, in the format "<value> <name> [<attr>
+// …​]". The value may be a hex sha1 hash, "@<dest>" for a symref, or
+// "?" to indicate that the helper could not get the value of the
+// ref. A space-separated list of attributes follows the name;
+// unrecognized attributes are ignored. The list ends with a blank
+// line.
+func (r *runner) handleList(ctx context.Context, args []string) (err error) {
 	if len(args) > 0 {
 		return errors.New("Lists for non-fetches unsupported for now")
 	}
 
-	_, err := r.initRepoIfNeeded(ctx)
+	repo, err := r.initRepoIfNeeded(ctx)
 	if err != nil {
 		return err
 	}
 
+	refs, err := repo.References()
```

Comment:
Does this include remote references or only local ones?

Reply:
The bare repo doesn't have any remote references.